proto/xudrpc.proto

```diff
@@ -500,7 +500,9 @@ message RemovePairResponse {}
 message ShutdownRequest {}
 message ShutdownResponse {}
 
-message SubscribeAddedOrdersRequest {}
+message SubscribeAddedOrdersRequest {
+  bool existing = 1 [json_name = "existing"];
```

Comment:
A comment here would be good, maybe "Whether to transmit all existing active orders upon establishing the stream".